torch_mlir/lib/aten_ops.cpp

```diff
@@ -0,0 +1,810 @@
+//===- aten_ops.cpp ---------------------------------------------*- C++ -*-===//
```

Comment:
No - this is in-tree to facilitate a working solution but is factored for ultimate contribution to PyTorch. Until the project evolves further, there is simply nowhere else this can feasibly exist. We've now clarified that in the various readmes.